routes/poll.js

```diff
@@ -0,0 +1,91 @@
+var express = require('express');
+var config = require('../config.json');
+var UserModel = require('../models/user');
+var QuestionModel = require('../models/question');
+var AnswerModel = require('../models/answer');
+const { db } = require('../models/user');
+const user = require('../models/user');
+//const user = require('../models/user');
+var router = express.Router();
+
+
+/* GET create page. */
+router.get('/create-poll', function(req, res) {
```

Comment:
Could we do just '/create' as the route name so that its not "poll/create-poll"?